build.cake

```diff
@@ -279,6 +279,10 @@ Task("Upload-AppVeyor-Artifacts")
 {
     AppVeyor.UploadArtifact(parameters.Paths.Files.ZipArtifactPathDesktop);
     AppVeyor.UploadArtifact(parameters.Paths.Files.ZipArtifactPathCoreClr);
+    foreach(var testResult in GetFiles(parameters.Paths.Directories.TestResults + "/*"))
```

Comment:
Yeah, that isn't going to work.  There are going to be multiple files in this folder, including the OpenCover result, which you don't really want to be uploading :-)